python/src/aiconfig/editor/server/server.py

```diff
@@ -65,7 +68,7 @@ def run_backend_server(edit_config: EditServerConfig) -> Result[str, str]:
             LOGGER.info("Initialized server state")
             debug = edit_config.server_mode in [ServerMode.DEBUG_BACKEND, ServerMode.DEBUG_SERVERS]
             LOGGER.info(f"Running in {edit_config.server_mode} mode")
-            app.run(port=edit_config.server_port, debug=debug, use_reloader=debug)
+            app.run(port=edit_config.server_port, debug=debug, use_reloader=debug, threaded=True)
```

Comment:
If the intention is to now allow concurrent requests, we should be exclusive-locking the aiconfig before doing any mutation. Otherwise you can get inconsistent states because the API does not support concurrent operations.